libgpiod.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# Instructions!
+# cd ~
+# wget https://raw.githubusercontent.com/adafruit/Raspberry-Pi-Installer-Scripts/master/libgpiod.sh
+# chmod +x libgpiod.sh
+# ./adafruit-pitft.sh
+
+
+echo "Installing build requirements - this may take a few minutes!"
+echo
+
+sudo apt-get update && sudo apt-get install -y \
+   autoconf \
+   autoconf-archive \
+   automake \
+   build-essential \
+   git \
+   libtool \
+   pkg-config \
+   python3 \
+   python3-dev \
+   python3-setuptools \
+   raspberrypi-kernel-headers \
+   swig3.0 \
+   wget
+
+build_dir=`mktemp -d /tmp/libgpiod.XXXX`
+echo "Cloning libgpiod repository to $build_dir"
+echo
+
+cd "$build_dir"
+git clone git://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git .
+
+echo "Building libgpiod"
+echo
+
+export PYTHON_VERSION=3
+./autogen.sh --enable-tools=yes --prefix=/usr/local/ --enable-bindings-python CFLAGS="-I/usr/include/python3.5/" \
```

Comment:
is there a way to get the current py3 include dir instead of hardcoding?

Reply:
See latest commit with `sysconfig`.